internal/auth/options.go

```diff
@@ -63,6 +63,7 @@ type Options struct {
 	EmailDomains     []string `envconfig:"SSO_EMAIL_DOMAIN"`
 	ProxyRootDomains []string `envconfig:"PROXY_ROOT_DOMAIN"`
 
+	AzureTenant              string `envconfig:"AZURE_TENANT"`
 	GoogleAdminEmail         string `envconfig:"GOOGLE_ADMIN_EMAIL"`
 	GoogleServiceAccountJSON string `envconfig:"GOOGLE_SERVICE_ACCOUNT_JSON"`
```

Comment:
Optional for wiring up OIDC as a provider:

```suggestion
	GoogleServiceAccountJSON string `envconfig:"GOOGLE_SERVICE_ACCOUNT_JSON"`
	OIDCDiscoveryURL         string `envconfig:"OIDC_DISCOVERY_URL"`
```